src/encoded/viewconfigs/auditview.py

```diff
@@ -28,16 +28,24 @@
 
 
 class AuditView(MatrixView):  #pylint: disable=too-few-public-methods
-    '''Audit View'''
-    _view_name = 'audit'
-    _factory_name = 'matrix'
+    """Audit View"""
+   
     def __init__(self, context, request):
-        super(AuditView, self).__init__(context, request)
+        """
+        Initialize.
+
+            :param context: Pyramid Context
+            :param request: Pyramid request
+        """   
+        super().__init__(context, request)
```

Comment:
@caseylitton: I did this because I was modifying the method and we were using verbose pre-Python 3.X style of using super.